CondTools/SiPixel/test/createTestDBObjects.sh

```diff
@@ -11,16 +11,16 @@ echo -e "TESTING Pixel 1D GenErr DB code ..."
 cmsRun ${SCRAM_TEST_PATH}/SiPixelGenErrorDBObjectUploader_cfg.py MagField=0.0 Fullname=SiPixelGenErrorDBObject_phase1_0T_mc_BoR3_v1_bugfix Map=${SCRAM_TEST_PATH}/../data/phaseI_mapping.csv GenErrFilePath=CalibTracker/SiPixelESProducers/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1 || die "Failure running SiPixelGenErrorDBObjectUploader_cfg.py" $?
 
 echo -e "TESTING Pixel 1D Template DB code for Phase-2 ..."
-cmsRun ${SCRAM_TEST_PATH}/SiPixelTemplateDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT615 Map=${SCRAM_TEST_PATH}/../data/phase2_T21_mapping.csv geometry=T21 TemplateFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v6.1.5_25x100_irradiated_v2_mc || die "Failure running SiPixelTemplateDBObjectUploader_Phase2_cfg.py" $?
+cmsRun ${SCRAM_TEST_PATH}/SiPixelTemplateDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT711 Map=${SCRAM_TEST_PATH}/../data/phase2_T33_mapping.csv geometry=T33 TemplateFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v7.1.1_25x100_v1_mc || die "Failure running SiPixelTemplateDBObjectUploader_Phase2_cfg.py" $?
 
 echo -e "TESTING Pixel 1D GenErr DB code for Phase-2 ..."
-cmsRun ${SCRAM_TEST_PATH}/SiPixelGenErrorDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT615 Map=${SCRAM_TEST_PATH}/../data/phase2_T21_mapping.csv geometry=T21 GenErrFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v6.1.5_25x100_irradiated_v2_mc || die "Failure running SiPixelGenErrorDBObjectUploader_Phase2_cfg.py" $?
+cmsRun ${SCRAM_TEST_PATH}/SiPixelGenErrorDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT711 Map=${SCRAM_TEST_PATH}/../data/phase2_T33_mapping.csv geometry=T33 GenErrFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v7.1.1_25x100_v1_mc || die "Failure running SiPixelGenErrorDBObjectUploader_Phase2_cfg.py" $?
 
-echo -e "TESTING Pixel 2D Template DB code for Phase-2 ..."
-cmsRun  ${SCRAM_TEST_PATH}/SiPixel2DTemplateDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT615 Map=${SCRAM_TEST_PATH}/../data/phase2_T21_mapping_den.csv TemplateFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v6.1.5_25x100_irradiated_v2_mc denominator=True || die "Failure running SiPixel2DTemplateDBObjectUploader_Phase2_cfg.py" $?
+#echo -e "TESTING Pixel 2D Template DB code for Phase-2 ..."
```

Comment:
Isn't this test portable to T33?

Reply:
No

Reply:
to elaborate further, the 2D templates are used in the IT digitizer to simulate radiation damage in the planar sensors (see https://github.com/cms-sw/cmssw/pull/40349 and https://github.com/cms-sw/cmssw/pull/40417). 
According to @tvami there is no 2D template payload available for the T33  (and above) tracker geometries, so apparently there weren't any irradiation scenarios considered (yet). 
I am keeping the line commented to re-instantiate the test when  at a later time an irradiation scenario becomes available for T33 (or for some subsequent - even final - tracker geometry)